Content.Server/Gatherable/Components/GatherableByHandComponent.cs

```diff
@@ -0,0 +1,21 @@
+namespace Content.Server.Gatherable.Components;
+
+[RegisterComponent]
+[Access(typeof(GatherableByHandSystem))]
+public sealed class GatherableByHandComponent : Component
```

Comment:
comment for the component and all datafields.

Reply:
oh, I forgot a comment for the component itself, I will add that in the next commit